ros2/test/pluginlib/BUILD.bazel

```diff
@@ -124,3 +125,29 @@ sh_test(
         ":py_loader_tests_binary",
     ],
 )
+
+ros2_ament_setup_library(
+    name = "triangle_ament_setup",
+    deps = [
+        ":triangle",
+    ],
+)
+
+ros2_ament_setup_library(
+    name = "square_ament_setup",
+    deps = [
+        ":square",
+    ],
+)
+
+cc_test(
+    name = "ament_setup_library_test",
```

Comment:
Please rename to ament_setup_library_tests and ament_setup_library_tests.cc, to be consistent with the rest of the repo.